docs/user/aws/customization.md

```diff
@@ -1,18 +1,50 @@
 # AWS Platform Customization
 
-The following options are available when using AWS:
+Beyond the [platform-agnostic `install-config.yaml` properties](../customization.md#platform-customization), the installer supports additional, AWS-specific properties.
 
-- `machines.platform.aws.rootVolume.iops` - the reserved IOPS of the root volume
-- `machines.platform.aws.rootVolume.size` - the size (in GiB) of the root volume
-- `machines.platform.aws.rootVolume.type` - the storage type of the root volume
-- `machines.platform.aws.type` - the EC2 instance type
-- `machines.platform.aws.zones` - a list of the availability zones that the installer will use when creating machines of this pool
-- `platform.aws.region` - the AWS region that the installer will use when creating resources
-- `platform.aws.userTags` - a map of keys and values that the installer will add as tags to all resources it creates
+## Cluster-scoped properties
+
+* `amiID` (optional string): The AMI that should be used to boot machines for the cluster.
+    If set, the AMI should belong to the same region as the cluster.
+* `region` (required string): The AWS region where the cluster will be created.
+* `userTags` (optional object): Additional keys and values that the installer will add as tags to all resources that it creates.
+    Resources created by the cluster itself may not include these tags.
+* `defaultMachinePlatform` (optional object): Default [AWS-specific machine pool properties](#machine-pools) which applies to [machine pools](../customization.md#machine-pools) that do not define their own AWS-specific properties.
+
+## Machine pools
```

Comment:
nit: match with something similar to `Cluster-scoped properties`

Reply:
> nit: match with something similar to `Cluster-scoped properties`

Do you mean "Machine pools" -> "Machine-pool properties"?  Or something else?